afids_utils/transforms.py

```diff
@@ -0,0 +1,32 @@
+"""Methods for transforming between different coordinate systems"""
+from __future__ import annotations
+
+from typing import Optional
+
+import numpy as np
+from numpy.typing import NDArray
+
+
+def afid_world2voxel(
+    afid_world: NDArray[np.float_],
+    nii_affine: NDArray[np.float_],
+    resample_size: float = 1,
+    padding: Optional[int] = None,
+) -> NDArray[np.int_]:
+    """Transform fiducials in world coordinates to voxel coordinates
+
+    Optionally, resample to match resampled image
```

Comment:
I think we should add some detail to the docstring about what `resample` is really doing (i.e. scaling the fiducial beyond what the affine did), and how (best) to use it -- it's not really clear to me, not having used it myself.

Reply:
I am leaning towards removing this as well since we don't do any image manipulation anymore and the resampling is pretty tied to resampled images.

For now I've just included a bit more detail in the docstring, though it still needs work. Which reminds me should start putting together docs for this as we build this up to and maybe I will try to tackle that after this PR is merged.